internal/server/v1/warden/errors.go

```diff
@@ -0,0 +1,8 @@
+package warden
+
+import "errors"
+
+var (
+	ErrTeamNotFound = errors.New("email is not registered on warden")
+	ErrUserNotFound = errors.New("user not authorized")
```

Comment:
`ErrUserNotFound` and `"user not authorized"` are two different cases, and why do we need this error?

another thing is, `ErrTeamNotFound` and `"email is not registered on warden"` are different meaning as well.